ui/app/components/secret-engine/configure-aws.hbs

```diff
@@ -17,12 +17,59 @@
     Access to AWS
   </h2>
   <div class="box is-fullwidth is-sideless">
-    <FormFieldGroups
-      @model={{@rootConfig}}
-      @mode={{if @rootConfig.isNew "create" "edit"}}
-      @modelValidations={{this.modelValidationsRoot}}
-      @useEnableInput={{true}}
-    />
+    {{! WIF is an enterprise only feature. We default to IAM access type for community users and display only those related form fields. }}
+    {{#if this.version.isEnterprise}}
+      <fieldset class="field form-fieldset" id="protection" data-test-access-type-section>
+        <legend class="is-label">Access Type</legend>
+        <p class="sub-text" data-test-access-type-subtext>
+          {{#if this.disableAccessType}}
+            You cannot edit Access Type if you have already saved access credentials.
+          {{else}}
+            Choose the way to configure access to AWS. Access can be configured either with IAM access keys, or using Plugin
+            Workload Identity Federation (WIF).{{/if}}</p>
+        <div>
+          <RadioButton
+            id="access-type-iam"
+            name="iam"
+            class="radio"
+            data-test-access-type="iam"
+            @value="iam"
+            @groupValue={{this.accessType}}
+            @onChange={{fn this.onChangeAccessType "iam"}}
+            @disabled={{this.disableAccessType}}
+          />
+          <label for="access-type-iam">IAM Credentials</label>
+
+          <RadioButton
+            id="access-type-wif"
+            name="wif"
+            class="radio has-left-margin-m"
+            data-test-access-type="wif"
+            @value="wif"
+            @groupValue={{this.accessType}}
+            @onChange={{fn this.onChangeAccessType "wif"}}
+            @disabled={{this.disableAccessType}}
+          />
+          <label for="access-type-wif">Workflow Identity Federation</label>
+        </div>
+      </fieldset>
+    {{/if}}
+    {{#if (eq this.accessType "wif")}}
+      <FormFieldGroups
+        @model={{@rootConfig}}
+        @mode={{if @rootConfig.isNew "create" "edit"}}
+        @modelValidations={{this.modelValidationsRoot}}
+        @groupName="fieldGroupsWif"
```

Comment:
Unsure why, but I tried to just render a conditional here and it borked all the things, so two separate code blocks it is. 

e.g. `@groupName={{if (eq this.accessType "wif" "fieldGroupsWif" "fieldGroupsIam"`